pkg/asset/installconfig/powervs/client.go

```diff
@@ -515,3 +516,24 @@ func (c *Client) GetAuthenticatorAPIKeyDetails(ctx context.Context) (*iamidentit
 func (c *Client) GetAPIKey() string {
 	return c.APIKey
 }
+
+// GetVPCs gets all VPCs in a region
+func (c *Client) GetVPCs(ctx context.Context, region string) ([]vpcv1.VPC, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	err := c.SetVPCServiceURLForRegion(ctx, region)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to set vpc api service url")
+	}
+
+	allVPCs := []vpcv1.VPC{}
+	if vpcs, detailedResponse, err := c.vpcAPI.ListVpcs(c.vpcAPI.NewListVpcsOptions()); err != nil {
+		if detailedResponse.GetStatusCode() != http.StatusNotFound {
+			return nil, err
+		}
```

Comment:
An error occurred, is there a reason to skip it here ?

Reply:
That check came from IBM Cloud code, and turns out it's rather misleading (giving the idea of potentially skipping), and in fact, it virtually never happens. So, I'm dropping the check in the next force-push. Thanks!!